udata/sentry.py

```diff
@@ -79,13 +50,9 @@ def init_app(app: UDataApp):
             log_level = getattr(logging, log_level_name.upper())
             sentry_sdk.set_level(log_level)
 
-        # Set sentry tags
-        tags = app.config["SENTRY_TAGS"]
-        for tag_key in tags:
-            sentry_sdk.set_tag(tag_key, tags[tag_key])
         # Versions Management: uData and plugins versions as tags.
         for dist in entrypoints.get_plugins_dists(app):
             if dist.version:
                 sentry_sdk.set_tag(dist.project_name, dist.version)
-        # Do not forget udata itself
-        sentry_sdk.set_tag("udata", pkg_resources.get_distribution("udata").version)
+        # Do not forget udata itself (is that necessary since we already have the release?)
```

Comment:
This might indeed not be needed

Reply:
Indeed I also think we can remove it. Done